### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by FirasH**
Fixed some typos in the script

### CODE REVIEW

File: 1-setup-linux-native.sh [hunk 1]
```diff
@@ -244,12 +244,12 @@ elif command -v pacman >/dev/null; then
             shared-mime-info \
             cmake make \
             python-lxml"
-    echo "Running pacman (you need root previllages to do that)..."
+    echo "Running pacman (you need root previleges to do that)..."
```
Comment (rodolforg) [inline 1-setup-linux-native.sh]: Shouldn't it be  "privileges"? ;)

Comment (FirasH) [reply 1-setup-linux-native.sh]: Yes you're right! My bad.

**Comment by FirasH** [comment]
> I think it's better you to rebase for squash this last commit into the first one, and force push it.

I have enabled "Allow squash merging", is that enough to squash the 2 commits as the pull request is merged?

**Comment by rodolforg** [comment]
I think it is :)

**Pull request merged**